lib/web_ui/test/golden_tests/golden_success_smoke_test.dart

```diff
@@ -6,10 +6,15 @@ import 'dart:html' as html;
 
 import 'package:test/test.dart';
 import 'package:ui/ui.dart';
+import 'package:ui/src/engine.dart';
 import 'package:web_engine_tester/golden_tester.dart';
 
-void main() {
+void main() async {
+  debugEmulateFlutterTesterEnvironment = true;
+  await webOnlyInitializePlatform(assetManager: WebOnlyMockAssetManager());
```

Comment:
(I guess there's some styles in the platform init that causes the color of the font to become red)

Reply:
Other than the above, LGTM!